content/en/blog/2024/hardening-the-collector-one.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Hardening the Collector #1 - A new default bind address'
+linkTitle: A new default bind address for the Collector
+date: 2024-07-02
+author: '[Pablo Baeyens](https://github.com/mx-psi) (OpenTelemetry, Datadog)'
+# prettier-ignore
+cSpell:ignore: awsfirehose awsproxy awsxray Baeyens jaegerremotesampling loki remotetap sapm signalfx skywalking splunk
+issue: 4760
+sig: Collector SIG
+---
+
+The OpenTelemetry Collector recently went through a security audit sponsored by
+the [CNCF](https://www.cncf.io/). As part of this process we recently published
+a security advisory related to a [DoS vulnerability](/blog/2024/cve-2024-36129/)
+that was fully addressed in v0.102.1.
+
+The security audit also motivated us to think about ways to harden official
+Collector builds and have a more secure default configuration. We are
+[actively][releases-586] [working][core-10469] [on several changes][core-10470]
+to achieve this and we will be publishing a series of blog posts to keep the
+community informed.
```

Comment:
As a reader, this leaves me wondering about the baseline or current security status of the Collector. How safe is it? Can we say?

Reply:
Would it help to link to https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md ? 

Or maybe, the security audit results would also be helpful here. I think we are going to publish them at some point, cc @jpkrohling

Reply:
I would add that those are recommendations, and not classified as issues or vulnerabilities. We can state that we are happy with the results, showing that secure coding practices are a solid part of our routine.

> While we expect the report to be made public soon, we can already say that we are very satisfied with the confirmation that the Collector has proven to be very secure, highlighting the secure coding practices and processes we have in place.

Reply:
> I would add that those are recommendations, and not classified as issues or vulnerabilities.

I went with 'security best practices'. I also added the sentence you mention :)